src/App.js

```diff
@@ -9,12 +12,16 @@ import './styles.css';
 class App extends Component {
   render() {
     return (
-      <BrowserRouter>
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route path="/login" component={Login} />
-        </Switch>
-      </BrowserRouter>
+      <div>
+        <MenuAppBar />
```

Comment:
Let's rearrange this so MenuAppBar/Drawer are inside the BrowserRouter component. This is because browser router passes props to its children automatically that give context to the current page. This is useful since we might be able to show the name of the current page in the menu bar based off of the url